frontend/src/container/GridGraphLayout/Graph/index.tsx

```diff
@@ -34,6 +35,22 @@ function GridCardGraph({
 	const { isAddWidget } = useSelector<AppState, DashboardReducer>(
 		(state) => state.dashboards,
 	);
+	const createDataset: Required<GetChartDataProps>['createDataset'] = (
+		element,
+		index,
+		allLabels,
+	) => ({
+		fill: true,
```

Comment:
config please also we are repeating here

Reply:
we adding the same config here in this one 

```
frontend/src/container/NewWidget/LeftContainer/WidgetGraph/WidgetGraph.tsx
```

Reply:
I see. I can try moving it to a common place. But besides that refactor, that config is needed in both places. This one for the widget, and the other one for the dashboard.